### PR TITLE
Fixes to support 1-click

### PR DESCRIPTION
**Pull request by DanRoseus**

### CODE REVIEW

File: products/bash/deploy-og-sub.sh [hunk 1]
```diff
@@ -29,9 +30,13 @@ function usage() {
 }
 
 namespace="cp4i"
+DEPLOY_DEMOS=false
 
-while getopts "n:p" opt; do
+while getopts "n:d" opt; do
   case ${opt} in
+  p)
```
Comment (James-Kirk) [inline products/bash/deploy-og-sub.sh:37]: Does this need to be `d` to match the arg?

File: products/bash/deploy-og-sub.sh [hunk 2]
```diff
@@ -29,9 +30,13 @@ function usage() {
...
+    DEPLOY_DEMOS=true
```
Comment (James-Kirk) [inline products/bash/deploy-og-sub.sh:38]: Where is this actually used? It seems to be set in this script but I can't see anywhere it's used?

File: products/bash/deploy-og-sub.sh (hunk 1 above)
Comment (DanRoseus) [reply products/bash/deploy-og-sub.sh:37]: Ooops, yes!

File: products/bash/deploy-og-sub.sh (hunk 2 above)
Comment (DanRoseus) [reply products/bash/deploy-og-sub.sh:38]: Should be used below:
```
if [[ "${DEPLOY_DEMOS}" == "true" ]]; then
```
should be:
```
if [[ "${USE_PRERELEASE_CATALOGS}" == "true" ]]; then
```

**Review by James-Kirk — APPROVED** [review]
LGTM

**Pull request merged**